packages/loader/container-loader/src/index.ts

```diff
@@ -21,3 +21,12 @@ export {
 } from "./location-redirection-utilities/index.js";
 export { IProtocolHandler, ProtocolHandlerBuilder } from "./protocol.js";
 export { tryParseCompatibleResolvedUrl, IParsedUrl } from "./utils.js";
+export {
+	IBaseProtocolHandler,
+	ProtocolOpHandler,
+	IScribeProtocolState,
+	IQuorumSnapshot,
+	Quorum,
+	QuorumClientsSnapshot,
+	QuorumProposalsSnapshot,
```

Comment:
I don't think most of these need to be exported. we probably only need to export IBaseProtocolHandler ProtocolOpHandler,